lily.toml

```diff
@@ -254,3 +254,9 @@ description = """
 Snapshot, pre-release, and release candidate builds, when available, can be found on GitHub at https://github.com/IrisShaders/Iris/releases. You can also compile a build yourself from https://github.com/IrisShaders/Iris/tree/1.18. Per rule 5, no support will be provided with compiling Iris. Builds will also occasionally be posted in <#883067831485366304>.  Snapshot, pre-release, and release candidate builds are also sometimes made available to Patreon supporters early.
 
  **Do not ask for an ETA on when Iris will update.**"""
+
+[[command]]
+name = "performance"
+help = "Informs the user about how to improve performance when running Minecraft"
+title = "Minecraft has extremely variable performance."
+description = "There's only a handful of things the support team can suggest to improve FPS/performance:\n\n- Optimize in-game settings (lower render distance, screen resolution etc.)\n- Install performance mods (<https://lambdaurora.dev/optifine_alternatives/>)\n- Ensure Minecraft is using the correct GPU (eg. dedicated instead of integrated)\n- Allocate an appropriate amount of RAM (4GB for 1.18+, 3GB for 1.17 and below)\n- Optimize system settings (power settings for Windows etc.)\n- Upgrade your hardware\n\nOther than the above advice, there is very little the support team can suggest to improve your performance. **If you're experiencing lower performance on newer versions, that's expected behaviour and there's nothing that can be done.**"
```

Comment:
Maybe put on the memory requirements a less than of equal too. We don't want to say that level of ram is 100% required

Reply:
Good call, added on both